alacritty/src/config/mod.rs

```diff
@@ -222,6 +222,12 @@ fn load_imports(config: &Value, config_paths: &mut Vec<PathBuf>, recursion_limit
             },
         };
 
+        if !path.exists() {
+            info!(target: LOG_TARGET_CONFIG, "Skipping importing config; not found:");
+            info!(target: LOG_TARGET_CONFIG, "  {}", path.display());
```

Comment:
```suggestion
            info!(target: LOG_TARGET_CONFIG, "  {:?}", path.display());
```

Reply:
`path.display()` returns an object that Implements the `Display` trait. Since it implements the display trait shouldn't we be calling the display format operator?

Reply:
The point is to automatically quote it, we do the same for regular config printing, so it's consistent.

Reply:
Ah ok I understand will do.